hapi-fees/src/main/java/com/hedera/services/usage/token/TokenBurnUsage.java

```diff
@@ -54,7 +60,8 @@ public FeeData get() {
 		var op = this.op.getTokenBurn();
 		if (currentSubType == SubType.TOKEN_NON_FUNGIBLE_UNIQUE) {
 			usageEstimator.addBpt((long) op.getSerialNumbersCount() * LONG_SIZE);
-			addTokenTransfersRecordRb(1, 0, op.getSerialNumbersCount());
+			serialNumsCount = op.getSerialNumbersCount();
+			addTokenTransfersRecordRb(1, 0, serialNumsCount);
```

Comment:
This is incorrect, we should use whatever `serialNumsCount` that was set.

Reply:
Afaict the code will get the correct result, but I agree it's more efficient to re-use the existing `serialNumCount` since the compiler doesn't seem able to inline method calls on gRPC object types. 👍

Reply:
@qnswirlds Used the `serialNumsCount` set in the class